include/motor-control/core/stepper_motor/motion_controller.hpp

```diff
@@ -206,6 +219,12 @@ class PipetteMotionController {
         queue.try_write(msg);
     }
 
+    [[nodiscard]] auto update_position(
```

Comment:
IMO to keep things consistent, we should treat this the same as the "encoder-position-ok-flag not set" case